addons/account/models/account_move.py

```diff
@@ -1129,10 +1105,7 @@ def _compute_amount(self):
             if move.is_invoice(include_receipts=True) and move.state == 'posted':
 
                 if currency.is_zero(move.amount_residual):
-                    if move.id in in_payment_set:
-                        new_pmt_state = 'in_payment'
-                    else:
-                        new_pmt_state = 'paid'
+                    new_pmt_state = 'paid'
```

Comment:
The `in_payment` state is now set by the `account_accountant` because "blabla split account/accountant modules blabla community/enterprise blabla group_account_user blablabla"

Reply:
then you should have defined a function in here _on_payment_paid_state() that returns 'paid' and that is overridden in enterprise to return 'in_payment', don't you think?

Reply:
To discuss